ray-on-gke/user/kuberay-tpu-webhook/main.go

```diff
@@ -0,0 +1,317 @@
+package main
```

Comment:
I personally recommend moving the `kuberay-tpu-webhook` directory to either:
* `ai-on-gke/ray-on-gke/kuberay-tpu-webhook` 
* `ai-on-gke/kuberay-tpu-webhook` 

I think `ray-on-gke/user` has been mostly for terraform modules and there were some discussions around removing it

Reply:
Done, moved to `ai-on-gke/ray-on-gke/kuberay-tpu-webhook`.